EIPS/eip-5585.md

```diff
@@ -0,0 +1,137 @@
+---
+eip: 5585
+title: EIP-721 NFT Authorization
+description: Allows NFT owners to authorize other users to use their NFTs.
+author: Veega Labs (@VeegaLabsOfficial), Sean NG (@ngveega), Tiger (@tiger0x), Fred (@apan), Fov Cao (@fovcao)
+discussions-to: https://ethereum-magicians.org/t/nft-authorization-erc721-extension/10661
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-15
+requires: 721
+---
+
+## Abstract
+
+This EIP separates an [EIP-721](./eip-721.md) NFT's ownership from its commercial usage rights to allow for the independent management of those rights.
+
+## Motivation
+
+Most NFTs have a simplified ownership verification mechanism, with a sole owner of an NFT. Under this model, other rights, such as exhibition, or derivative works, are not possible to grant, limiting the value and commercialization of NFTs. Therefore, the separation of an NFT's ownership and user rights can enhance its commercial value. 
+
+With the development of the Metaverse, NFTs are becoming more diverse, with new use cases such as digital collections, virtual real estate, and commercial copyrights. Meanwhile, the commercial rights, such as exhibition, derivative or commercial authorization based on NFTs are becoming a potential business form.  
```

Comment:
There are competing ERCs addressing the ability to rent out, share or use temporarily NFTs, as well as the privilege managements. I'd suggest you also try to demonstrate why this standard is better than the competing ERCs in the Motivation and Rationale of design choices

Reply:
Hi, @xinbenlv Thanks for your suggestion. So basically commercial authorization is a broader concept including many kinds of right and renting is just one of them.

Reply:
Sounds good, I suggest you put that into Motivation section. Also in ratinale consider mention the unique design choice you made that create such merit

Reply:
Not a problem, thanks @xinbenlv

Reply:
> Sounds good, I suggest you put that into Motivation section. Also in ratinale consider mention the unique design choice you made that create such merit

Hi, @xinbenlv , we have added those into the EIP and created a pull request based on 5585, thanks for your suggestions again.